modules/library/src/main/scala/zio/elasticsearch/ElasticQuery.scala

```diff
@@ -87,6 +87,20 @@ object ElasticQuery {
   final def contains(field: String, value: String): WildcardQuery[Any] =
     Wildcard(field = field, value = s"*$value*", boost = None, caseInsensitive = None)
 
+  /**
+   * Constructs an instance of [[zio.elasticsearch.query.DisjunctionMax]] using the specified parameters.
+   *
+   * @param queries
+   *   the type-safe [[ElasticQuery]] object to be wrapped inside of disjunction max query
+   * @tparam S
+   *   document for which field query is executed
+   * @return
+   *   an instance of [[zio.elasticsearch.query.DisjunctionMax]] that represents the `disjunction max` query to be
+   *   performed.
+   */
+  final def disjunctionMax[S](queries: Chunk[ElasticQuery[S]]): DisjunctionMaxQuery[S] =
```

Comment:
Can you implement raw method also? You can look at `bool` query as a reference (`filter` methods for example)

Reply:
Also, check if `[S: Schema]` is required.

Reply:
Can there be zero queries? If so, you can use ElasticQuery[S]* maybe.

Reply:
I think that there must be at least one query, as it is a required field.